docs/Checkpoints.md

```diff
@@ -0,0 +1,49 @@
+Checkpoints
+===
+
+There are 3 types of OLMo checkpoints.
+
+1. OLMo (standard) checkpoints. These checkpoints can be produced and used by the code in this repo. "OLMo checkpoints" will typically refer to these checkpoints.
+2. Transformers checkpoints. These checkpoints can be produced and used via the OLMo implementation in the [Hugging Face Transformers](https://huggingface.co/docs/transformers/index) library. As we continue to develop and improve OLMo, our implementation in this repo may temporariy become incompatible with the implementation in the Transformer library.
+3. HF OLMo checkpoints. These checkpoints can be produced and used via the `hf_olmo` package. The `hf_olmo` package provides basic Transformers functionality while always staying compatible with the OLMo library.
+
+OLMo (standard) checkpoints
+---
+
+There are 2 categories of OLMo checkpoints:
+- unsharded: a complete checkpoint in a standard form;
+- sharded: a checkpoint that has been broken down into smaller components, for easier use in our multi-node training.
+
+Unless otherwise specified, an OLMo checkpoint is assumed to be unsharded. OLMo sharded and unsharded checkpoints can run the pretraining and fine-tuning provided in this repo.
```

Comment:
Might be more clear this way
```suggestion
Unless otherwise specified, an OLMo checkpoint is assumed to be unsharded. OLMo sharded and unsharded checkpoints can be used with the pretraining/fine-tuning script provided in this repo.
```